bolt-simple-taproot.md

```diff
@@ -769,21 +766,22 @@ The to remote output has the following form:
   * `OP_1 to_remote_output_key`
   * where:
     * `taproot_nums_point = 0245b18183a06ee58228f07d9716f0f121cd194e4d924b037522503a7160432f15`
-    * `to_remote_output_key = taproot_nums_point + tagged_hash("TapTweak", taproot_nums_point|| to_remote_script_root`
+    * `to_remote_output_key = taproot_nums_point + tagged_hash("TapTweak", taproot_nums_point || to_remote_script_root)`
     * `to_remote_script_root = tapscript_root([to_remote_script])`
     * `to_remote_script` is the remote script:
         ```
-        <remotepubkey> OP_CHECKSIGVERIFY 1 OP_CHECKSEQUENCEVERIFY
+        <remotepubkey> OP_CHECKSIG
+        OP_CHECKSEQUENCEVERIFY
```

Comment:
Clever...but will need to double check the semantics here to ensure this doesn't become an any-one-can-spend...

Reply:
Confirmed that it all checks out: https://github.com/Roasbeef/lightning-rfc/pull/1#issuecomment-1170686197